### PR TITLE
Adds Grunt Config and Multiple Package Support

### PR DESCRIPTION
**Pull request by losandes**
Hi,

First of all - thanks for this package. I needed to be able to audit packages that were not in the immediate folder, and to audit multiple packages at once, so I added config and multi-package support. I believe I maintained all of the default behaviors and that this is not a breaking change. Let me know if you have any questions, or want me to make some changes.

Thanks,

&y

### CODE REVIEW

**Comment by bear** [comment]
@losandes looks like we may have to update some package.json packages as our own nsp test is failing :)

Can you do that as part of this PR? or do you want us to do that and then you can resubmit?

**Comment by losandes** [comment]
Sure, I can do that.

**Comment by losandes** [comment]
There have been 7 breaking releases of the npm-registry-client since this was written, so this will take some effort. I'll get back to you soon - I might have some questions.

**Comment by bear** [comment]
ouch! thanks for doing this and yes, feel free to ask any questions you have and I'll find the right person to answer them.

**Comment by losandes** [comment]
OK @bear. I ended up refactoring it to leverage the nsp package, so there is less redundant code between the projects, and this one should be easier to maintain. Looks like it makes the badges happy :+1:. 

I changed some of the terminology (all documented in the README), but I'm fairly certain there are no breaking changes - the previous terminology is maintained.

**Comment by bear** [comment]
Yesterday we briefly entertained the thought of suggesting that you should take advantage of NSP but figured that could be done later... and now you've done it!

thanks!

**Comment by bear** [comment]
This is +1 for my non-JS look, but I want @nlf to give it a once over before we merge it.

thanks again!

**Comment by losandes** [comment]
Sounds good. I saw some reference to heaving nsp start using requireSafe. If that's in the plans give me a heads up and I can help keep this up to date.

**Comment by losandes** [comment]
oh - assuming you accept the PR and merge it. Will you guys take care of doing an `npm publish`?

**Comment by bear** [comment]
@losandes yes, we will publish it

File: tasks/package.js [hunk 1]
```diff
@@ -1,180 +1,205 @@
...
-
+var grunt = require('grunt'),
+    auditPackage = require('nsp/lib/auditPackage.js'),
+    path = require('path'),
+    async = require('async'),
+    chalk = require('chalk'),
+    table = require('text-table');
 
 module.exports = function (grunt) {
+    var makeConfig,
```
Comment (nlf) [inline tasks/package.js]: declare the vars as they're used, please. it helps make refactoring simpler.

**Comment by nlf** [comment]
aside from the one comment i left, this looks really good. once that's been addressed i think go ahead and merge this. thanks @losandes!

**Comment by losandes** [comment]
changes are in

**Comment by bear** [comment]
thanks @losandes - I'll merge this and get it on the "npm publish" list in a bit

**Pull request merged**